inventory/byo/hosts.byo.grafana.example

```diff
@@ -0,0 +1,14 @@
+[OSEv3:children]
+masters
+nodes
+
+[OSEv3:vars]
+ansible_ssh_user=root
+deployment_type=origin
+gf_datasource_name="example"
```

Comment:
Possibly just document these options in inventory/hosts.example.

Reply:
right, rebase was required.
will merge those things, 10x

Reply:
Please update inventory/hosts.example with a section for Grafana configuration.
For example:
```
# Grafana Configuration
#gf_datasource_name="example"
#gf_prometheus_namespace="openshift-metrics"
#gf_oauth=true
```